pkg/aws/errors/ec2_errors.go

```diff
@@ -0,0 +1,6 @@
+package errors
+
+const (
+	NotFoundAssociationID = "InvalidAssociationID.NotFound"
```

Comment:
nit: seems this is not used anywhere, can we delete it?

Reply:
This will be need for adding disassociating call.